### ISSUE TITLE
NVDA not Working to edit Documents in PJE When Using Chrome.

### ISSUE THREAD

**Issue by Marlin86** [nvaccess/nvda#17376]
Expected result: NVDA should be able to interact with editable frames. 
Result: when I open the document, the field does not appears.

PJE is an app used in Brazil to work with judicial process.
Previously, NVDA worked greatly with it, both using Chrome or Firefox.
At the moment, just can be used with Firefox.
Using Crhome, when I open a document in a frame edit, the text edit field doesn't appears. I don't no how I can show the problem to you, since I need to sign with a id document and password.

**Comment by seanbudd** [comment]
Welcome @Marlin86 ,

We are unable to process this issue as it stands.

- Please fill out the [issue template fully.](https://github.com/nvaccess/nvda/issues/new?assignees=&labels=&projects=&template=bug_report.md)
- Please reproduce this issue and provide a [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in general preferences.
- This is very likely a problem with the implementation of the application/website, as a starting point please submit feedback to the authors to let them know about the problem.

Kind Regards,
NV Access Software Developers
